zebra-state/src/service/finalized_state/disk_format/transparent.rs

```diff
@@ -242,7 +243,7 @@ impl AddressBalanceLocation {
         &mut self,
         unspent_output: &transparent::Output,
     ) -> Result<(), amount::Error> {
-        self.balance = (self.balance + unspent_output.value()).unwrap_or(MAX_MONEY.try_into()?);
+        self.balance = ( self.balance +  Amount::<MaxInt64>::try_from(i64::from(unspent_output.value())) )?; 
```

Comment:
```suggestion
self.balance = (self.balance + unspent_output.value().constrain::<MaxInt64>())?;
```
May be use of `constrain` looks better for conversion between different constraints of Amount? Non-principal, but just an offer.

Reply:
'constrain' fits better here, 
thanks